src/impulsoetl/sisab/relatorio_validacao_producao/principal.py

```diff
@@ -80,12 +82,23 @@ def obter_validacao_producao(
                         ficha=ficha,
                         aplicacao=aplicacao,
                         )
-                    teste_validacao(df_extraido=df_extraido,df_tratado=df_tratado)
-                    carregar_dados(
-                        sessao = sessao, 
-                        df_tratado = df_tratado,
-                        tabela_destino = tabela_destino,
-                        periodo_id = periodo_id,
-                        no_prazo = envio_prazo,
-                        ficha_tipo = ficha,
-                        aplicacao_tipo = aplicacao)
+                    verificar_relatorio_validacao_producao(df_extraido=df_extraido,df_tratado=df_tratado)
+                    df_consolidado = pd.concat([df_consolidado, df_tratado], ignore_index=True)
+
+                    logger.info(
+                    "Captura realizada e tratada para a ficha de `{ficha}`"
+                    + "com aplicação '{aplicacao}' no período '{periodo_codigo}'"
+                    + "e envio no prazo = {envio_prazo}",
+                    ficha=ficha,
+                    aplicacao=aplicacao,
+                    periodo_codigo=periodo_codigo,
+                    envio_prazo=envio_prazo,
+                    )
+        
+            carregar_dados(
```

Comment:
*Acho* que a ideia era deixar esta linha alinhada com o primeiro `for` (linha 58), não? Como está agora, você carrega o dataframe uma vez para envio no prazo, e outra para todos os envios; não sei se era essa a ideia

Reply:
Antes estava realizando cargas para cada aplicação, ficha. Isso não era necessário, pois nem havia tanto volume. Procurei carregar para cada envio prazo, pois percebi que esse relatório é muito mais instável e alguma consulta de ficha há possibilidade de cair a página. O prioritário é ter o envio prazo=false no banco. Por isso consolidei o dataframe depois desse _for_